reco_utils/recommender/deeprec/io/dkn_iterator.py

```diff
@@ -62,6 +62,24 @@ def __init__(self, hparams, graph, col_spliter=" ", ID_spliter="%"):
             self.click_news_entity_values = tf.placeholder(
                 tf.int64, [None], name="click_news_entity"
             )
+        self.news_word_index = {}
+        self.news_entity_index = {}
+        with tf.gfile.GFile(hparams.news_feature_file, "r") as rd:
+            while True:
+                line = rd.readline()
+                if not line:
+                    break
```

Comment:
I think we can simply:
`for line in rd:`

instead of
```
while True:
    line = rd.readline()
    if not line:
        break
```

Reply:
fixed, good catch!